README.adoc

```diff
@@ -162,16 +162,16 @@ include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClien
 ----
 This class is a client for the `system` service.
 
-In this class there is a method called `getPropertiesHelper()`. This method makes a request to the
-`system` service, and checks for the response. If a `Status.OK`` response which is corresponding to the
+A method called `getPropertiesHelper()` is included in this class. This method makes a request to the
+`system` service and checks for the response. If a `Status.OK` response that corresponds to the
```

Comment:
How about this to fix the passive voice? "If the request responses a `Status.OK` that corresponds to the 200 status code, the method returns the system properties from the `system` service request."

Reply:
How about this? "If the request response is a `Status.OK` message that corresponds to the 200 status code, the method returns the system properties from the `system` service request."